src/foundry_dev_tools/cli/config.py

```diff
@@ -0,0 +1,90 @@
+"""The configuration CLI.
+
+Allows to view and edit the Foundry DevTools configuration.
+"""
+
+from __future__ import annotations
+
+import os
+import platform
+import sys
+from typing import TYPE_CHECKING
+
+import click
+from rich.console import Console
+from rich.prompt import IntPrompt
+
+from foundry_dev_tools.cli.info import _config_section
+from foundry_dev_tools.utils.config import find_project_config_file, site_cfg_file, user_cfg_files
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+if sys.version_info < (3, 11):
+    pass
+else:
+    pass
+
+
```

Comment:
Add migrate method fdt config migrate, that
a) migrate to toml
b) rename old config.